### PR TITLE
Update: If a table footer is present add the "column-index" data attr…

### PR DESCRIPTION
**Pull request by Dgc2002**
…ibute for use in other plugins

### CODE REVIEW

**Comment by Dgc2002** [comment]
While working with the ColResize plugin I ran into an issue where I was having to do some  pretty hacky things to enable column resizing using the table footer as well as header. ColResize takes advantage of the fact that ColReorder adds the ```column-index``` data attribute to header cells. This PR simply extends that functionality to table footer elements as well.  
  
Here's an example of how it currently behaves. You are able to resize a column using the table header but not the footer. [Demo](http://codepen.io/Dgc2002/pen/grOKpj?editors=1010)  
 
Here is an example of the result of this PR. You are able to resize the column using both header and footer elements: [Demo](http://codepen.io/Dgc2002/pen/Vawdey)  
  
I've only made a hand full of pull requests to established repos. Let me know if I'm missing anything.

**Comment by Dgc2002** [comment]
Sometimes using Git is like herding cats. That got messy, I can resubmit a straight pull request if you'd like.

**Comment by DataTables** [comment]
Thanks for the PR. I see the issue that this resolves, however, DataTables allows the footer elements to be a bit more flexible than the header in that a cell can colspan multiple columns.

I'm not sure how that should be handled here (multiples column indexes for a cell, or skip indexes?). This probably needs some thought "up stream" in DataTables about how to handle this.

**Comment by Dgc2002** [comment]
Ahh I see, I wasn't aware of that.  
  
I think skipping indexes might obscure the meaning of the property.  
  
For my immediate application I could see something like a ```column-index-start-X``` and ```column-index-end-X``` working. I was thinking ```column-index-range-X-Y``` but that's starting to get a bit wordy.  
  
I'll just pull this functionality into ColResize for the time being.

**Comment by AllanJard** [comment]
ColReorder 2 is a total rewrite and is a lot cleaner in how it moves the columns around. Going to close as this PR is old (my fault - sorry).
